src/fedora_easyfix/utilities/consumer.py

```diff
@@ -0,0 +1,143 @@
+"""
+    Copyright (c) 2021 Fedora Websites and Apps
+
+    Permission is hereby granted, free of charge, to any person obtaining a copy
+    of this software and associated documentation files (the "Software"), to deal
+    in the Software without restriction, including without limitation the rights
+    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+    copies of the Software, and to permit persons to whom the Software is
+    furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in all
+    copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+    SOFTWARE.
+"""
+
+from json import loads
+from time import time
+
+
+class ErraticReturns:
+    def parameter_error_return_data(self):
+        return_data = {
+            "status": "FAIL",
+            "salute": "Could not fetch requested information",
+            "reason": "Command string could not be interpreted",
+            "collection_fetched_at": time()
```

Comment:
This is mostly a matter of taste: `collection_fetched_at` is pretty long, I'd just call it e.g. `time`. I'm also not sure if everything your API returns below is a "collection", so if it should convey information that this value isn't about the object(s) in question but when it was/they were retrieved, you could call it e.g. `time_of_retrieval`.